src/components/Match/MatchContainer.tsx

```diff
@@ -50,7 +84,7 @@ const initialState = {
   orderedSuggestions: []
 };
 
-const reducer = ( state, action ) => {
+const reducer = ( state: StateType, action: ActionType ): StateType => {
```

Comment:
I think it's okay to call this State and Action, like [here](https://github.com/inaturalist/iNaturalistReactNative/blob/aab6bf6aa163a859b59c3a32ec8f29d12d151b0c/src/providers/ExploreContext.tsx#L389-L390) if you want.